### ISSUE TITLE
Voice conversion training ?

### ISSUE THREAD

**Issue by skol101** [sh-lee-prml/HierSpeechpp#28]
Hello, thanks for posting TTS(TTV) training code!

Are you planning to releae VC training instructions , too?

**Comment by sh-lee-prml** [comment]
Yes. we have a plan to release the VC training code after paper notification... (Sorry for that we have not received any responses..!)

Now, we are trying to change the train code with real-time pitch extractor for efficient large-scale dataset because YAAPT is too slow...!

If we confirm the stability for real-time pitch extractor, we may release the code a little earlier (than paper notification!)

Thanks!

**Comment by platform-kit** [comment]
@sh-lee-prml that's great news! Can you provide a rough ETA for either scenario? IE are you expecting either to occur in the next 6 months? 3 months? And will the license be permissive (i.e. allowing commercial use?)

**Comment by nellorebhanuteja** [comment]
@sh-lee-prml 
any update on paper acceptance, and training code availability?
